src/NuGet.Core/NuGet.CommandLine.XPlat/external/docs.md

```diff
@@ -0,0 +1,364 @@
+    
+
+---
+title: dotnet nuget add source command
+description: The `dotnet nuget add source` command will add a new source to your NuGet configuration files. This will enable finding packages at this additional source. 
```

Comment:
What do you think of this?

```suggestion
description: The `dotnet nuget add source` command adds a new package source to your NuGet configuration files.
```

The two sentences are a little similar, hence why I merged them.

Reply:
yep. made the change.